bin/ddtrace

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
```

Comment:
is `ddtrace` the name of the program installed?
It'd be better to have something like `ddtracerb` so it does not conflict with other languages.

Reply:
`ddtrace` is the name of the gem installed, yeah. Intuitively it makes sense to use this name, but you might be right... if there's another `ddtrace` on the load path, it might not run as it should.

Reply:
Changed this name to `ddtracerb`